contrib/dropwizard/src/main/java/io/opencensus/contrib/dropwizard/DropWizardMetrics.java

```diff
@@ -0,0 +1,339 @@
+    TimeSeries timeSeries =
+        TimeSeries.create(
+            Collections.<LabelValue>emptyList(),
+            Collections.singletonList(Point.create(value, clock.now())),
+            null);
+    return Metric.create(
+        metricDescriptor, Collections.unmodifiableList(Collections.singletonList(timeSeries)));
+  }
+
+  /**
+   * Returns a {@code Metric} collected from {@link Counter}.
+   *
+   * @param dropwizardName the metric name.
+   * @param counter the counter object to collect.
+   * @return a {@code Metric}.
+   */
+  private Metric collectCounter(String dropwizardName, Counter counter) {
+    String metricName = DropWizardUtils.generateFullMetricName(dropwizardName, "counter");
+    String metricDescription =
+        DropWizardUtils.generateFullMetricDescription(dropwizardName, counter);
+
+    MetricDescriptor metricDescriptor =
+        MetricDescriptor.create(
+            metricName,
+            metricDescription,
+            DEFAULT_UNIT,
+            Type.GAUGE_INT64,
+            Collections.<LabelKey>emptyList());
+    TimeSeries timeSeries =
+        TimeSeries.create(
+            Collections.<LabelValue>emptyList(),
+            Collections.singletonList(
+                Point.create(Value.longValue(counter.getCount()), clock.now())),
+            null);
+    return Metric.create(
+        metricDescriptor, Collections.unmodifiableList(Collections.singletonList(timeSeries)));
+  }
+
+  /**
+   * Returns a list of {@link Metric}s collected from {@link Meter}.
+   *
+   * @param dropwizardName the metric name.
+   * @param meter the meter object to collect
+   * @return a list of {@link Metric}s.
+   */
+  private List<Metric> collectMeter(String dropwizardName, Meter meter) {
+    String metricName = DropWizardUtils.generateFullMetricName(dropwizardName, "meter");
+    String metricDescription = DropWizardUtils.generateFullMetricDescription(dropwizardName, meter);
+
+    MetricDescriptor countMetricDescriptor =
+        MetricDescriptor.create(
+            metricName,
+            metricDescription,
+            DEFAULT_UNIT,
+            Type.CUMULATIVE_INT64,
+            Collections.<LabelKey>emptyList());
+    TimeSeries countTimeSeries =
+        TimeSeries.create(
+            Collections.<LabelValue>emptyList(),
+            Collections.singletonList(Point.create(Value.longValue(meter.getCount()), clock.now())),
+            null);
+
+    // Collect rate related metric
+    metricName = DropWizardUtils.generateFullMetricName(dropwizardName, "meter_rate");
+    metricDescription = DropWizardUtils.generateFullMetricDescription(dropwizardName, meter);
+
+    MetricDescriptor rateMetricDescriptor =
+        MetricDescriptor.create(
+            metricName,
+            metricDescription,
+            "events/second",
+            Type.CUMULATIVE_DOUBLE,
+            Collections.unmodifiableList(Collections.singletonList(RATE_LABEL_KEY)));
```

Comment:
This "Collections.unmodifiableList(Collections.singletonList(RATE_LABEL_KEY))" is a constant. Same for "Collections.singletonList(RATE_MEAN_LABEL_VALUE)"

Reply:
Done